.github/workflows/sync.yml

```
@@ -0,0 +1,65 @@
+      - name: List Files
+        run: |
+          echo "Checked out the following files for update:"
+          find data -type f
+      - name: Pull Data
+        run: make pull-data
+      - name: Commit Changes
+        uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          commit_message: "chore: sync data with remote sources"
+      - name: "Run if changes have been detected"
+        if: steps.auto-commit-action.outputs.changes_detected == 'true'
```

actionlint (1.7.9)

property "auto-commit-action" is not defined in object type {determine_updates: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)